src/arch/ucx/machine.C

```diff
@@ -32,7 +32,7 @@
 #define CmiSetMsgSize(msg, sz)    ((((CmiMsgHeaderBasic *)msg)->size) = (sz))
 
 #define UCX_MSG_PROBE_THRESH            32768
-#define UCX_MSG_NUM_RX_REQS             64
+#define UCX_MSG_NUM_RX_REQS             16
```

Comment:
Any specific reason for choosing 16 for the number of pre-posted buffers?

Reply:
I made some experiments with p2p benchmarks and looks like it gives slightly better performance

Reply:
I was wondering if having more buffers is useful for an "all-to-all" sort of situation, but since we've provided a switch (`+ucx_num_rx_reqs`) for controlling this, I think we should be good.